scene/3d/path_3d.cpp

```diff
@@ -133,6 +133,8 @@ void Path3D::_curve_changed() {
 			PathFollow3D *child = Object::cast_to<PathFollow3D>(get_child(i));
 			if (child) {
 				child->update_configuration_warnings();
+				// Also update the transform of the PathFollow3Ds.
```

Comment:
Comment is probably unnecessary here, code is simple enough.

Reply:
fixed